src/scss/components/layout/_index.scss

```diff
@@ -136,7 +135,7 @@ body.hxHorizontal {
   /* Legacy Grid Implementation */
   display: -ms-grid;
   -ms-grid-columns: 1fr;
-  -ms-grid-rows: @header-height 1fr @footer-height;
+  -ms-grid-rows: $header-height 1fr $footer-height;
 
   /* Modern Grid Implementation */
```

Comment:
Please update all single line comments to use the double slash comment `//` syntax for the entire file.

Reply:
Please check the changes in latest commit